server/lib/webhook.js

```diff
@@ -62,8 +64,14 @@ function webhook(
       }
       const res = {
         payload,
-        error: errorInfo
+        hull_summary: "",
+        error: errorInfo,
+        status: error.status
       };
+      if (error.status >= 300 && error.status <= 308) {
+        res.hull_summary =
+          "Server returned a redirect code - see `Connector doesn't support redirects` in documentation";
```

Comment:
It might generate a lot of error notifications and not update the status on the connector. 
I see that we never check the endpoint or its response in the connector status check. What about updating the connector status when we see that we get a 300 error to mark the connectors as in error in the connectors list